packages/flutter_tools/test/general.shard/github_template_test.dart

```diff
@@ -16,8 +16,8 @@ import '../src/common.dart';
 import '../src/context.dart';
 
 void main() {
-  BufferLogger logger;
-  FileSystem fs;
+  late BufferLogger logger;
+  FileSystem? fs;
```

Comment:
`late`
I'm running out of steam here, please audit the rest 🙂

Reply:
Done